R/aglm.R

```diff
@@ -36,6 +37,7 @@ aglm <- function(x, y,
                  qualitative_vars_OD_only=NULL,
                  quantitative_vars=NULL,
                  use_LVar=FALSE,
+                 extrapolation="default",
```

Comment:
It's OK here, but would fix the quote marks around default value of `OD_type_of_quantitatives`, from single quotes to double quotes to keep consistency.